frontend/src/components/Output/ResponsePlot.js

```diff
@@ -1,5 +1,37 @@
-import React from "react";
-const ResponsePlot = props => {
-    return <div id="model_chart"></div>;
-};
+import React, {Component} from "react";
+import {inject, observer} from "mobx-react";
+import _ from "lodash";
+import Plotly from "plotly.js";
+
+@inject("outputStore")
+@observer
+class ResponsePlot extends Component {
+    componentDidMount() {
```

Comment:
the code looks nearly  identical between DidMount and DidUpdate; does it make sense to create a new method that both can use?

Reply:
created a createScatterPlot() function